### ISSUE TITLE
Config for Kowlski on staging is wrong

### ISSUE THREAD

**Issue by guynir42** [skyportal/skyportal#3206]
The port and host address for kowalski in the config file in staging is different from that in the secrets file. 
This probably has to do with the way we combine config files, that needs to be clarified.

**Comment by stefanv** [comment]
Yes, ww make custom patches to the different configs in the deploy.py script.

**Comment by Theodlz** [comment]
Not sure we should even have Kowalski in fritz's repo, I mean we don't run it next to Fritz for a long time now, maybe we shouldn't keep it there its unnecessary and confusing.

**Comment by mcoughlin** [comment]
No more Kowalski.
